docs/Guide/utilities/Discord Utilities/UsefulRegexes.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 0
+title: Useful Regexes
+---
+
+## Introduction
+
+Sapphire provides a lot of Regexes that you can use in your code
+
+::: note
+
+If you don't know what a Regex is check the
+[Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)
+
+:::
+
+### Channel Mention Regex
+
+This regex will match any mention of a channel and return the id of the channel
```

Comment:
Add the raw of each regex under each header.

I.e.

---

The raw regex string of this regex is as follows:
```typescript
raw regex here
```

---

No tabs are needed for this. It's the same in everything anyway.